llrt_utils/src/encoding.rs

```diff
@@ -73,6 +76,7 @@ impl Encoder {
             Self::Hex => bytes_from_hex(&bytes),
             Self::Base64 => bytes_from_b64(&bytes),
             Self::Utf8 | Self::Iso88591 => Ok(bytes),
+            Self::Utf16le => Err(String::from("This method is not yet supported")),
```

Comment:
Forgot to implement?

Reply:
I did not forget it, but I did not implement it because it was not needed in Text[En|De]coder and I was not confident in implementing it correctly.

Please check back with me once I have fixed it.